apps/site/src/app/(home)/sections/Sponsors/Sponsors.module.scss

```diff
@@ -8,61 +8,68 @@
 
 .title {
 	text-align: center;
-}
-
-.clipboard {
-	position: relative;
-	padding: 64px 32px 48px 32px;
-	min-height: 512px;
-	width: 80%;
-	margin: 0 auto;
-	border: 20px solid #aa703c;
-	border-radius: 100px;
-	background: #ffffff;
-	display: flex;
-	flex-direction: column;
-	align-items: stretch;
+	color: theme.$white;
+	font-size: 3rem;
+	font-weight: bold;
+	margin-bottom: 3rem;
 
-	@include utils.media-breakpoint-up(sm) {
-		padding: 32px 32px 48px 32px;
+	@include utils.media-breakpoint-down(sm) {
+		font-size: 2rem;
 	}
 }
 
-.clip {
-	height: 192px;
-	position: absolute;
-	left: 50%;
-	transform: translateX(-50%) rotate(90deg);
-	top: -105px;
-
-	@include utils.media-breakpoint-up(sm) {
-		height: 256px;
-		top: 50%;
-		transform: translateY(-50%);
-		left: -85px;
-	}
+.logos {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	gap: 1.5rem;
+	max-width: 1200px;
```

Comment:
Suggestion: rather than manually constraining the inner elements, we can use Bootstrap's native container sizing (unfortunately not done last year) by changing the main `<section>` element to be a `Container`, see Home Intro ([`Intro.tsx:14`](https://github.com/HackAtUCI/zothacks-site/blob/main/apps/site/src/app/(home)/sections/Intro/Intro.tsx#L14)) for example.